docs/topics_5/proc_web-obtaining-latest-image.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide_5/master.adoc
+
+
+[id="proc_web-obtaining-latest-image_{context}"]
+= Obtaining the latest image
+
+The first step in troubleshooting in an OpenShift Container Platform environment is to be sure you are using the latest image.
+
+NOTE: If you have deployed the {WebName} by pasting in a JSON template, no image stream is created and the following steps are not applicable.
+
+.Procedure
+
+. Open the OpenShift console.
+. Click *Builds* -> *Images*.
+. Ensure that the column under *Tags* indicates that each image is on the `latest` stream.
+. If the latest image is not in use, follow the steps for the OpenShift installation to redeploy the latest image of the {WebName}.
```

Comment:
Passive voice. Change to "If the image is not the latest version".

Again, does this installation process that they have to do refer only to the Operator installation? In that case, say so. Telling the user to follow the installation steps is not helpful if there are 3 installation procedures for OpenShift.

Reply:
@jonathanvila does this installation process that they have to do refer only to the Operator installation?

Reply:
@RichardHoch no, the operator takes care of which version to install ( for the v.0.0.1 it will always install "stable" tag ), so the user should not have to deal with images versions

Reply:
@jonathanvila So should we remove "obtaining the latest image" from troubleshooting? It sounds like a user won't have this problem.

Reply:
@RichardHoch regarding the operator yes you could remove this section for the operator.